target/linux/airoha/patches-5.15/0005-spi-Add-support-for-the-Airoha-EN7523-SoC-SPI-contro.patch

```diff
@@ -166,7 +166,7 @@
 +	}
 +}
 +
-+static void set_spi_clock_speed(int freq_mhz)
++/*static void set_spi_clock_speed(int freq_mhz)
```

Comment:
I would prefer `#if 0` to comment such block.

Reply:
What I used to comment stuff was using the shit comment format of `//`